docs/local_development_kube.md

```diff
@@ -10,116 +10,48 @@ It's like having a conversation with your cluster.
 
 This guide assumes you have the following tools installed:
 
+- [brew](https://brew.sh/) if you are on linux
```

Comment:
I did not need brew on Linux. Do you mean MacOS?
I was hoping someone on MacOS could write a dedicated part about the setup.

Reply:
Didn't you ? Well, I'm on Ubuntu and I there is at least one of these tools (helmfile, I think) is not Debian-friendly at all. Do you remember how you installed it ? Or are you running it as a container ?

I believe @Morendil is also trying to set this up on Mac. Maybe he can pitch in.

Reply:
The helmfile I used (in docker) is described in the documentation you propose to remove 😅